### DISCUSSION TITLE
Show text when there are no multiplayer lounge rooms available

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#16599] [Ideas]
i think it would be nice to have some text that notifies the user that there are no available rooms at the moment just like osu stable.
![osu_2022-01-24_23-12-35](https://user-images.githubusercontent.com/43358824/150874743-4c96fb6b-3400-473d-8570-41d21dbb45bd.png)
![screenshot279](https://user-images.githubusercontent.com/43358824/150874762-c57bb050-bcca-49c2-8119-12a860b55028.jpg)

**Comment by peppy** [comment]
It would also be helpful to show if other rulesets *do* have games available, for cases where the user is looking at the wrong ruleset (or just interested in potentially playing a different one).

**Reply by Sublimelime** [reply]
How should this be shown? I was thinking of basically copying stable and having 

> There are no rooms available.

> Click 'Create room' to create a room!

text in the center of the screen (like stable). If other rulesets have games, what would be the best way to show that on the UI? Additional text saying "Other rulesets have active games!", highlighting the other rulesets in some way (flash the top left icons that have active games?), something else?

**Reply by peppy** [reply]
I think it's fine to show text for now. Might even be able to reuse the `NotFoundDrawable` from `BeatmapListingOverlay`.
